docker/bootstrap/Dockerfile.common

```diff
@@ -1,5 +1,8 @@
 FROM golang:1.5
 
+# Use all cores for parallel make (load bound to number of cores as well).
+ENV MAKEFLAGS -j$(nproc) -l$(nproc)
```

Comment:
I think this hard-codes the $(nproc) value at the time of building the image, and that value will take effect for everything you do with that image on any machine later. I'd prefer that this env var be set only for the install_grpc.sh script - either within the script, or at the start of the RUN directive.